### ISSUE TITLE
Seems not running

### ISSUE THREAD

**Issue by schimits221** [ppy/osu#3304]
I've download and install lastest build but it seems not running and not making any logs.
![untitled](https://user-images.githubusercontent.com/42657530/44553624-37cde080-a758-11e8-8add-c6c7c00fabb9.png)

**Comment by schimits221** [comment]
yes, i found with same solution at github. so my excuse are lazer won't
open without the lastest of NET.Framework as you said.

**Comment by peppy** [comment]
lazer no longer depends on a .net framework (it comes bundled) for what it's worth.

**Comment by schimits221** [comment]
That's a good news! but here i can't search where the song was stored if i
have not installing Osu!Stable.
